src/mappings/omim.sssom.tsv

```diff
@@ -7826,6 +7825,7 @@ MONDO:0015855		skos:exactMatch	omim.ps:113700	Breasts and/or nipples, aplasia or
 MONDO:0015912		skos:exactMatch	OMIM:155100	macrothrombocytopenia and granulocyte inclusions with or without nephritis or sensorineural hearing loss	semapv:UnspecifiedMatching
 MONDO:0015942		skos:exactMatch	omim.ps:305620	Frontometaphyseal dysplasia	semapv:UnspecifiedMatching
 MONDO:0015977		skos:exactMatch	omim.ps:601495	Agammaglobulinemia	semapv:UnspecifiedMatching
+MONDO:0015991		skos:exactMatch	omim.ps:215700	Citrullinemia	semapv:UnspecifiedMatching
```

Comment:
## _Outdated_: Strange; shouldn't show certain changes

I noticed some strangeness, but before I could comment, I had to refresh, and now the issue is gone, so no worries.

<details><summary>Original comments</summary>
<p>

### `fix_mim.ru` shouldn't be added
Your base for this PR is the `issue-560` branch, which already includes this file, so I'm not sure why it's showing as new here too.

And nothing shows up when I do `git diff origin/issue-560-BUILD-3 origin/issue-560 --name-only | grep fix_mim`.

### `fix_make_omim_exact.ru` and `fix_omimps.ru` shouldn't have any changes
There's no diff between these files and your other branch either.

---

So, looks like a GitHub display error for these.

</p>
</details>